packages/core/template/default/404.md

```diff
@@ -0,0 +1,4 @@
+<br>
+
+# 404 page not found
+You can edit this file to create your own 404 page.
```

Comment:
Perhaps as a default, we can provide a "go back"/ “home" button.
As for the note on editing, perhaps can be more specific and say "edit the 404.md file" to customize the 404 page?